import/_icons.scss

```diff
@@ -1,4 +1,5 @@
 @import "../src/icons/icon.functions";
 @import "../src/icons/icon.mixins";
 @import "../src/icons/icon.variables";
+@import "../src/icons/generated-icons-map";
```

Comment:
As mentioned: eg. `generated-icons-map` -> `icon-map.GENERATED` to make it even more obvious that this is an auto generated file.